src/Abp.Web.Mvc/Web/Mvc/Extensions/HtmlHelperResourceExtensions.cs

```diff
@@ -101,8 +103,8 @@ private static string GetPathWithVersioningForEmbeddedFile(string path)
             {
                 embeddedResourcePath = embeddedResourcePath.Substring(1);                
             }
-
-            var resource = WebResourceHelper.GetEmbeddedResource(embeddedResourcePath);
+            
```

Comment:
Please revert undesirable whitespace.

Reply:
The whitespace here is preserved from old line 104.  I can trim it up, but should I then also trim old line 99 and 94 so it's not the only if without a padded line in the method?

Reply:
You can see from [the files diff](https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5191/commits/270c5d8dcd350f10d1b51415b8ce1b34b34705bb#diff-6262387c5950d3c2f56c37a6d0dd5bb3) and [the raw file](https://raw.githubusercontent.com/aspnetboilerplate/aspnetboilerplate/270c5d8dcd350f10d1b51415b8ce1b34b34705bb/src/Abp.Web.Mvc/Web/Mvc/Extensions/HtmlHelperResourceExtensions.cs) on GitHub that the whitespace is added, rather than preserved.

Old lines 99 and 94 already do not have whitespace.

Reply:
I'm not sure why the diff highlights two lines, both the whitespace line and the var resource line as a subtract, and then shows them both as an add.  The resource line was a change and the whitespace is the same in both changesets.

The raw file does show whitespace, but it is the same whitespace as the original file.  

Here's what I see both on a raw side-by-side and just looking at what's currently in dev:
https://imgur.com/a/BUXqcvR

Like I said, I'm happy to take out the whitespace I just want to be sure we're on the same page before I do.

Reply:
The diff highlights two lines because you modified both.

Ah, I think you misunderstand. I am talking about whitespace, not empty lines.
Old lines 94, 99 and 104 are empty lines without whitespace.

Please leave the empty lines and take out the whitespace.

Reply:
Ah, I see now.  Thank you for the feedback.  Corrected!